src/components/SnippetAreaToolbar.jsx

```diff
@@ -99,36 +105,43 @@ const SnippetAreaToolbar = (props) => {
           onChange={onLanguageChange}
           style={styles.toolbarField}
         />
-        <div>
+        <div style={styles.buttonsContainer}>
           <LockButton
             onClick={onLockClick}
             readOnly={readOnly}
-            style={styles.buttons}
+            style={styles.button}
           />
           <SaveMenu
-            canSave={canSave}
+            canSave={canEdit}
```

Comment:
Originally 'canSave' contained the permissions for whether a user could save a snippet or not. The same permissions are needed to determine whether a user can delete a snippet, which is why it was changed to 'canEdit' as this seemed to be more generic naming. Do you have a different suggestion? @solkaz